### PR TITLE
Glib -> 2.74.1

### PR DESCRIPTION
**Pull request by satmandu**
Builds properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=glib2741 CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/glib.rb [hunk 1]
```diff
@@ -33,12 +33,13 @@ class Glib < Package
   depends_on 'util_linux' # R
   depends_on 'zlibpkg' # R
   depends_on 'pcre2' # R
-  no_env_options
+
+  no_strip if ARCH == 'armv7l' || ARCH == 'aarch64'
```
Comment (uberhacker) [inline packages/glib.rb:37]:
```
no_strip if ARCH.include?['aarch64', 'armv7l']
```
?

File: packages/glib.rb [hunk 2]
```diff
@@ -33,12 +33,13 @@ class Glib < Package
   depends_on 'util_linux' # R
   depends_on 'zlibpkg' # R
   depends_on 'pcre2' # R
-  no_env_options
+
+  no_strip if ARCH.include['aarch64', 'armv7l']
```
Comment (uberhacker) [inline packages/glib.rb:37]: Sorry, I meant:
```
no_strip if ['aarch64', 'armv7l'].include?ARCH
```
Don't forget the `?` too.

Comment (satmandu) [reply packages/glib.rb]: Done!

Comment (uberhacker) [reply packages/glib.rb]: You could probably do:
```
no_strip if %w[aarch64 armv7l].include?ARCH
```

**Review by uberhacker — APPROVED** [review]

**Pull request merged**